pages/docs/reference/building-mpp-with-gradle.md

```diff
@@ -1885,13 +1843,23 @@ binaries.findExecutable("foo", DEBUG)
 </div>
 </div>
 
+> Note: Prior to 1.3.40 both test and product executables were represented by the same binary type. Thus to access the default test binary created by the plugin, the following snipped was used:
+> ```
+> binaries.getExecutable("test", "DEBUG")
+> ``` 
+> Since 1.3.40 test executables are represented by a separate binary type and have their own getter. To access the default test binary, use:
```

Comment:
```suggestion
> Since 1.3.40, test executables are represented by a separate binary type and have their own getter. To access the default test binary, use:
```